### DISCUSSION TITLE
Keycloak operator - watch all namespaces

### DISCUSSION THREAD

**Discussion by mitko577** [keycloak/keycloak#13449] [Q&A] [answered]
Hello, 

I am trying to run the othe perator in Kubernetes cluster and to watch all namespaces. Even when I add:
          env:
            - name: WATCH_NAMESPACE
              value: ""
The operator does not create Keycloaks. 

Any suggestions, how to run the operator and watch all namespaces will be apriciated.

Thanks in advance!

**Comment by Jamstah** [comment, marked as answer]
See https://github.com/keycloak/keycloak/issues/15888 for current progress
